Description: undefine G_DISABLE_DEPRECATED where needed
  undefine G_DISABLE_DEPRECATED in files that use g_format_size_for_display
  or G_CONST_RETURN (maybe other stuff too but those are the errors I
  Spotted when making the patch)
Bug-Debian: http://bugs.debian.org/649645

--- evolution-3.0.3.orig/widgets/table/gal-a11y-e-cell.c
+++ evolution-3.0.3/widgets/table/gal-a11y-e-cell.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 
 #include <string.h>
--- evolution-3.0.3.orig/widgets/table/gal-a11y-e-table-item.c
+++ evolution-3.0.3/widgets/table/gal-a11y-e-table-item.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 
 #include <string.h>
--- evolution-3.0.3.orig/widgets/table/gal-a11y-e-table-click-to-add.c
+++ evolution-3.0.3/widgets/table/gal-a11y-e-table-click-to-add.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 
 #include <atk/atk.h>
--- evolution-3.0.3.orig/widgets/table/gal-a11y-e-table-column-header.c
+++ evolution-3.0.3/widgets/table/gal-a11y-e-table-column-header.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 #include <glib/gi18n.h>
 #include <atk/atkobject.h>
--- evolution-3.0.3.orig/widgets/misc/ea-cell-table.c
+++ evolution-3.0.3/widgets/misc/ea-cell-table.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-cell-table.h"
 
 EaCellTable *
--- evolution-3.0.3.orig/widgets/misc/ea-calendar-item.c
+++ evolution-3.0.3/widgets/misc/ea-calendar-item.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <stdio.h>
 #include <time.h>
 #include <string.h>
--- evolution-3.0.3.orig/widgets/misc/e-picture-gallery.c
+++ evolution-3.0.3/widgets/misc/e-picture-gallery.c
@@ -19,6 +19,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "e-util/e-icon-factory.h"
 
 #include "e-picture-gallery.h"
--- evolution-3.0.3.orig/widgets/misc/e-attachment-paned.c
+++ evolution-3.0.3/widgets/misc/e-attachment-paned.c
@@ -19,6 +19,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "e-attachment-paned.h"
 
 #include <glib/gi18n.h>
--- evolution-3.0.3.orig/widgets/misc/e-attachment-tree-view.c
+++ evolution-3.0.3/widgets/misc/e-attachment-tree-view.c
@@ -19,6 +19,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "e-attachment-tree-view.h"
 
 #include <glib/gi18n.h>
--- evolution-3.0.3.orig/widgets/misc/e-attachment.c
+++ evolution-3.0.3/widgets/misc/e-attachment.c
@@ -19,6 +19,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "e-attachment.h"
 
 #include <errno.h>
--- evolution-3.0.3.orig/widgets/misc/ea-calendar-cell.c
+++ evolution-3.0.3/widgets/misc/ea-calendar-cell.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <gtk/gtk.h>
 #include <e-util/e-util.h>
 #include "ea-calendar-cell.h"
--- evolution-3.0.3.orig/calendar/gui/ea-jump-button.c
+++ evolution-3.0.3/calendar/gui/ea-jump-button.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-jump-button.h"
 #include "ea-calendar-helpers.h"
 #include "ea-week-view.h"
--- evolution-3.0.3.orig/calendar/gui/ea-week-view.c
+++ evolution-3.0.3/calendar/gui/ea-week-view.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-week-view.h"
 #include "ea-cal-view-event.h"
 #include "ea-calendar-helpers.h"
--- evolution-3.0.3.orig/calendar/gui/ea-week-view-main-item.c
+++ evolution-3.0.3/calendar/gui/ea-week-view-main-item.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-week-view-main-item.h"
 #include "ea-week-view.h"
 #include "ea-week-view-cell.h"
--- evolution-3.0.3.orig/calendar/gui/ea-day-view.c
+++ evolution-3.0.3/calendar/gui/ea-day-view.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-day-view.h"
 #include "ea-cal-view-event.h"
 
--- evolution-3.0.3.orig/calendar/gui/ea-gnome-calendar.c
+++ evolution-3.0.3/calendar/gui/ea-gnome-calendar.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-gnome-calendar.h"
 #include "e-calendar-view.h"
 #include <string.h>
--- evolution-3.0.3.orig/calendar/gui/ea-cal-view-event.c
+++ evolution-3.0.3/calendar/gui/ea-cal-view-event.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-cal-view-event.h"
 #include "ea-calendar-helpers.h"
 #include "ea-day-view.h"
--- evolution-3.0.3.orig/calendar/gui/ea-day-view-cell.c
+++ evolution-3.0.3/calendar/gui/ea-day-view-cell.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-day-view-cell.h"
 #include "ea-day-view-main-item.h"
 #include "ea-day-view.h"
--- evolution-3.0.3.orig/calendar/gui/ea-week-view-cell.c
+++ evolution-3.0.3/calendar/gui/ea-week-view-cell.c
@@ -22,6 +22,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-week-view-cell.h"
 #include "ea-week-view-main-item.h"
 #include "a11y/ea-factory.h"
--- evolution-3.0.3.orig/calendar/gui/ea-cal-view.c
+++ evolution-3.0.3/calendar/gui/ea-cal-view.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-cal-view.h"
 #include "ea-calendar-helpers.h"
 #include "e-day-view.h"
--- evolution-3.0.3.orig/calendar/gui/ea-day-view-main-item.c
+++ evolution-3.0.3/calendar/gui/ea-day-view-main-item.c
@@ -21,6 +21,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "ea-day-view-main-item.h"
 #include "e-day-view-top-item.h"
 #include "ea-day-view.h"
--- evolution-3.0.3.orig/mail/e-mail-attachment-bar.c
+++ evolution-3.0.3/mail/e-mail-attachment-bar.c
@@ -19,6 +19,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "e-mail-attachment-bar.h"
 
 #include <glib/gi18n.h>
--- evolution-3.0.3.orig/mail/mail-mt.c
+++ evolution-3.0.3/mail/mail-mt.c
@@ -15,6 +15,8 @@
  * Copyright (C) 1999-2008 Novell, Inc. (www.novell.com)
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 
 #include <stdio.h>
--- evolution-3.0.3.orig/mail/e-mail-store.c
+++ evolution-3.0.3/mail/e-mail-store.c
@@ -19,6 +19,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include "e-mail-store.h"
 
 #include <glib/gi18n.h>
--- evolution-3.0.3.orig/addressbook/gui/widgets/ea-addressbook-view.c
+++ evolution-3.0.3/addressbook/gui/widgets/ea-addressbook-view.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 
 #include <glib/gi18n.h>
--- evolution-3.0.3.orig/addressbook/gui/widgets/ea-minicard.c
+++ evolution-3.0.3/addressbook/gui/widgets/ea-minicard.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 #include <string.h>
 #include <glib/gi18n.h>
--- evolution-3.0.3.orig/addressbook/gui/widgets/ea-minicard-view.c
+++ evolution-3.0.3/addressbook/gui/widgets/ea-minicard-view.c
@@ -20,6 +20,8 @@
  *
  */
 
+#undef G_DISABLE_DEPRECATED
+
 #include <config.h>
 #include <string.h>
 #include <gtk/gtk.h>
